### PR TITLE
Fix : Shapes SVG include PHP Parse Error elementor#9693

### PR DESCRIPTION
**Pull request by MarieComet**
## PR Checklist

- [ x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

* This PR propose a fix for the issue about PHP Parse Error explained here https://github.com/elementor/elementor/issues/9693

## Test instructions
This PR can be tested by following these steps:

* Add a custom SVG shape, the SVG file must begin with <?xml version="1.0" encoding="UTF-8"?>
For example :
`<?xml version="1.0" encoding="UTF-8"?> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 111.859 500 42.943"><path d="M.366 156.606h498.56v-45.71L.366 156.606z" fill="#bada55"></path></svg>`

## Quality assurance

- [x ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: includes/elements/section.php [hunk 1]
```diff
@@ -1530,7 +1530,7 @@ private function print_shape_divider( $side ) {
 		$negative = ! empty( $settings[ $base_setting_key . '_negative' ] );
 		?>
```
Comment (bainternet) [inline includes/elements/section.php]: Looks good but we need an extra check if the file actually exists something like 
```suggestion
		$shape_path = Shapes::get_shape_path( $settings[ $base_setting_key ], $negative );
		if ( ! file_exists( $shape_path ) ) {
			return;
		}
		?>
```

File: includes/elements/section.php [hunk 2]
```diff
@@ -1530,7 +1530,7 @@ private function print_shape_divider( $side ) {
...
 		<div class="elementor-shape elementor-shape-<?php echo esc_attr( $side ); ?>" data-negative="<?php echo var_export( $negative ); ?>">
-			<?php include Shapes::get_shape_path( $settings[ $base_setting_key ], ! empty( $settings[ $base_setting_key . '_negative' ] ) ); ?>
+			<?php echo file_get_contents( Shapes::get_shape_path( $settings[ $base_setting_key ], ! empty( $settings[ $base_setting_key . '_negative' ] ) ) ); ?>
```
Comment (bainternet) [inline includes/elements/section.php]: and then this can be done like this:
```suggestion
			<?php echo file_get_contents( $shape_path ); ?>
```

**Review by bainternet — CHANGES_REQUESTED** [review]
Thanks for the PR, 
can you please correct the requested changes.

**Comment by MarieComet** [comment]
Thanks for the review @bainternet 
New PR with requested changes : https://github.com/elementor/elementor/pull/9734

**Comment by bainternet** [comment]
closed in favor of #9734

**Pull request closed without merging**